apiary.apib

```diff
@@ -167,38 +191,83 @@ This action response with the queried material data.
 
 + Response 200 (application/json)
 
-        {
-            "data": {
-                "attributes": {
-                    "name": "sample abc"
-                },
-                "id": "8b8959b0-e852-0133-db51-005056bf12f5",
-                "relationships": {
-                    "material-type": {
-                        "data": {
-                            "id": "1",
-                            "type": "material-types"
-                        }
-                    },
-                    "metadata": {
-                        "data": []
-                    }
-                },
-                "type": "materials"
-            },
-            "included": [
-                {
-                    "attributes": {
-                        "name": "sample"
-                    },
-                    "id": "1",
-                    "links": {
-                        "self": "/api/v1/material_types/1"
-                    },
-                    "type": "material-types"
-                }
-            ]
-        }
+       {
+         "data": {
+           "id": "75ed7330-ee86-0133-bcfd-685b35d4beab",
+           "type": "materials",
+           "attributes": {
+             "name": "library 123"
+           },
+           "relationships": {
+             "material-type": {
+               "data": {
+                 "id": "2",
+                 "type": "material-types"
+               }
+             },
+             "metadata": {
+               "data": [
+                 {
+                   "id": "4",
+                   "type": "metadata"
+                 },
+                 {
+                   "id": "5",
+                   "type": "metadata"
+                 }
+               ]
+             },
+             "parents": {
+               "data": [
+                 {
+                   "id": "a53a07f0-ee83-0133-bcfa-685b35d4beab",
+                   "type": "materials"
```

Comment:
Just a thought: We may want to add link tags to the parents and the children, but then probably it would be in the included section